### PR TITLE
Navigation bar layout

### PR DESCRIPTION
**Pull request by j3rryzh**

### CODE REVIEW

File: react-app/src/NavBar.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+      <Text style={styles.text}>Relief in Seat</Text>
+    </View>
+  )
+}
+
+const styles = StyleSheet.create({
+  container: {
+    flexDirection: 'row',
+    // flex: 1,
```
Comment (mylesieong) [inline react-app/src/NavBar.js:31]: Do we need this?

File: react-app/src/NavBar.js [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+    height: 24,
+    width: 24,
+    margin: 14,
+    paddingHorizontal: 3,
+    paddingVertical: 6,
+    justifyContent: 'space-between',
+  },
+  text: {
+    color: '#FFFFFF',
+    // fontFamily: 'Inter',
```
Comment (mylesieong) [inline react-app/src/NavBar.js:47]: Do we need this?

File: react-app/src/NavBar.js (hunk 1 above)
Comment (huishi329) [reply react-app/src/NavBar.js:31]: Yes, as far as I know, we need this to style our app

**Review by huishi329 — APPROVED** [review]

**Review by huishi329 — APPROVED** [review]

File: react-app/src/NavBar.js [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+      <Pressable
+        onPress={() => Alert.alert('This is a recall function.')}
+        style={styles.button}
+      >
+        <View style={styles.bar}></View>
+        <View style={styles.bar}></View>
+        <View style={styles.bar}></View>
+      </Pressable>
+
+      <Text style={styles.text}>Relief in Seat</Text>
```
Comment (mylesieong) [inline react-app/src/NavBar.js:23]: Lets put the name "Home"

Comment (j3rryzh) [reply react-app/src/NavBar.js:23]: You meant the file name or what else?

**Review by mylesieong — APPROVED** [review]

**Pull request merged**